### PR TITLE
Fade dr

### PR DESCRIPTION
**Pull request by oisinod**
adding this conduit to show dr values

### CODE REVIEW

File: analysis/priest/src/TranslucentImage.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  constructor(options: Options) {
+    super(options);
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.TRANSLUCENT_IMAGE.id);
+
+    if (!this.conduitRank) {
+      this.active = false;
+      return;
+    }
+    this.conduitIncrease = TRANSLUCENT_IMAGE_INCREASE[this.conduitRank];
```
Comment (emallson) [inline analysis/priest/src/TranslucentImage.tsx]: this should use `parser/core/conduitScaling` with the appropriate rank 1 value (which based on the other file is i think `6`?) unless the scaling doesn't follow regular conduit scaling (`rankOneValue * (1 + (rank - 1) * 0.1)`)

File: analysis/priestdiscipline/src/constants.ts [hunk 2]
```diff
@@ -138,3 +138,21 @@ export const SWIFT_PENITENCE_INCREASE = [
...
+  9,
+  9.6,
+  10.2,
+  10.8,
+  11.4,
+  12,
+  12.6,
+  13.2,
+  13.8,
+];
```
Comment (emallson) [inline analysis/priestdiscipline/src/constants.ts]: this is unnecessary if you use `conduitScaling`

**Review by emallson — CHANGES_REQUESTED** [review]
small change for conduit scaling.

i also merged the main branch in to catch it up with some other changes

**Comment by oisinod** [comment]
should have all those changes addressed

**Review by emallson — APPROVED** [review]

**Pull request merged**